browse/services/document/author_affil.py

```diff
@@ -270,18 +283,21 @@ def _parse_author_affil_back_propagate(author_list: List[List[str]],
     return author_list
 
 
-def split_authors(authors: str)->List:
-    """Split author string into authors entity lists.
+def split_authors(authors: str) -> List:
```

Comment:
Not a comment on the changes in this PR. But, this is a fairly complicated bit of parsing going on that might be simplified by 1) writing a very simple recursive descent parser, 2) not parsing in the future, if we no longer work with author lists. Hopefully (2), as I'm not sure recursive descent is an easy match for python, without looking into it more.